src/meta/proto-conv/src/data_from_to_protobuf_impl.rs

```diff
@@ -491,6 +491,7 @@ impl FromToProto for dv::IntervalKind {
     where Self: Sized {
         let dv_kind = match p {
             pb::IntervalKind::Year => dv::IntervalKind::Year,
+            pb::IntervalKind::Quarter => dv::IntervalKind::Quarter,
```

Comment:
IMHO, adding another enum item makes the message incompatible:
`Quoter` is 8, when an old program loads a new message, it will fail to load it and return an error:

https://github.com/datafuselabs/databend/blob/2dfb0a2b3236cc861e81633cf67dcb94e3d25eec/src/meta/proto-conv/src/data_from_to_protobuf_impl.rs#L520-L529

Enum item is not like struct message field. An unknown struct field will be just ignored.

Reply:
![God_please_no](https://user-images.githubusercontent.com/7812909/187735455-507f76c7-5efe-4fce-abcc-c1c32454f9f3.gif)

Reply:
This is an incompatible feature. It looks like there is no other option.

The choice is to leave it as is, which would be a silent incompatible change, or to increase `MIN_COMPATIBLE_VER` to `VER` to mark it as incompatible explicitly.

Reply:
When upgrading a query cluster, some new query nodes may start feeding new version data into the store, and the old version query nodes have a chance to consume the new version data. 

Thus compatibility is maintained in bi-direction, between query-query or query-meta or meta-meta.

The changes in this PR kind of break the contract of `VER` and `MIN_COMPATIBLE_VER`: compatible versions of programs should be running together forever without any trouble.

But if such a short-term incompatible state is not a thing to worry about while upgrading a cluster, I'd be cool with it.

Reply:
Got it, I'll be happy to ignore this state.  Let's have this `short-term incompatible` now.